src/utils/signedUrlReplacer.ts

```diff
@@ -36,6 +33,16 @@ export function getFilePathFromUrl(url: string) {
   }
 }
 
+export function getAbsoluteFileName(filePath: string): string | null {
+  try {
+    const parts = filePath.split('/')
+    return parts.pop() || null
```

Comment:
As long as the filePath is a string `parts.pop()` should always return a value right?
The only case when `parts.pop()` will be falsy is when the parts array is empty, or the last element in array is `undefined`, `null`, `0`, `false`, `""` etc. Is there a case this can occur?